rust/agents/relayer/src/msg/pending_message.rs

```diff
@@ -300,6 +300,28 @@ impl PendingOperation for PendingMessage {
 }
 
 impl PendingMessage {
+    /// Constructor that tries reading the retry count from the HyperlaneDB in order to recompute the `next_attempt_after`.
+    /// In case of failure, behaves like `Self::new(...)`.
+    pub fn from_persisted_retries(message: HyperlaneMessage, ctx: Arc<MessageContext>) -> Self {
+        let mut pm = Self::new(message, ctx);
+        if let Some(num_retries) = pm
+            .ctx
+            .origin_db
+            .retrieve_pending_message_data_by_message_id(&message.id())
+        {
+            let next_attempt_after =
+                PendingMessage::calculate_msg_backoff(num_retries).map(|dur| Instant::now() + dur);
+            pm.num_retries = num_retries;
+            pm.next_attempt_after = next_attempt_after;
+        } else {
+            info!(
```

Comment:
I'd suggest making this a trace (or maybe debug) - iiuc this log will be made whenever the relayer tries to process a new message that it didn't know from a previous restart, which is a very common occurrence. People tend to get spooked by messages with the word "failed" in it, so maybe also worth rephrasing as "No persisted retry count for message {}" or something

Reply:
Good point, turned that into a trace